SQLBiblioteca.sql

```diff
@@ -0,0 +1,49 @@
+create database Db_biblioteca;
+
+CREATE TABLE autor (
+    id_autor int identity,
+    name_autor varchar(255),    
+    primary key (id_autor)
+);
+
+CREATE TABLE seccion (
+    id_seccion int identity,
+    name_seccion varchar(255),    
+    primary key (id_seccion)
+);
+
+
+CREATE TABLE biblioteca (
+    id_biblioteca int identity,
+    name_biblioteca varchar(255),    
+    primary key (id_biblioteca)
+);
+
+CREATE TABLE libro (
+    id_libro int identity,
+    edicion Decimal,
+	a�o date,
```

Comment:
Please avoid using ñ, instead of use anio

Reply:
has been correct